libs/designsystem/src/lib/components/modal/footer/modal-footer.component.ts

```diff
@@ -1,8 +1,32 @@
-import { Component } from '@angular/core';
+import { Component, Input, OnInit, ElementRef, OnChanges, SimpleChanges } from '@angular/core';
 
 @Component({
   selector: 'kirby-modal-footer',
   templateUrl: './modal-footer.component.html',
   styleUrls: ['./modal-footer.component.scss'],
 })
-export class ModalFooterComponent {}
+export class ModalFooterComponent implements OnInit, OnChanges {
+  @Input() snapToKeyboard = false;
```

Comment:
Consider if it's cleaner to style through `@HostBinding`:
```suggestion
  @HostBinding('class.snap-to-keyboard') @Input() snapToKeyboard = false;
```